volatility3/framework/layers/registry.py

```diff
@@ -96,6 +95,41 @@ def __init__(
                 f"Exception when setting hive {self.name} max address, using {hex(self._maxaddr)}",
             )
 
+    def _find_registry_process(self) -> Optional["extensions.EPROCESS"]:
+        """Walk the active process list and return the Registry process if it exists. Duplicates
+        PsList.list_processes() since pulling in the plugin causes problems.
+
+        Returns:
+            The Registry EPROCESS object if it exists, or None
+        """
+
+        kernel = self.context.modules.get("kernel")
```

Comment:
```suggestion
        kernel = self.context.modules.get(self.config["kernel_module_name"])
```
This shouldn't be hard coded as `"kernel"`, there's no guarantee what it'll be called. So this should be possible to pull from the config (as that's how it used to work).  I don't _think_ there's a nicer way of getting to it, but at least this won't need to import pslist now...